test/e2e/helpers.js

```diff
@@ -7,14 +7,16 @@ const { difference } = require('lodash');
 const WebSocket = require('ws');
```

Comment:
This file has logic changes in withFixtures file:
- accept localNode argument: to choose between which client(s) we want to use 'ganache', 'anvil'...
- accept localNodeOptions: to pass options whichever node(s) we want to use
- removing disableGanace argument: now we can just signal that in localNode: 'none'
- not yet removing, but adding support to handle multiple clients, by passing arrays into the node and options, like ', ['anvil', 'anvil'] -- I've migrate 1 spec to use this. In subsequent PR we can remove the doubleGanache logic completely and migrate all specs